### PR TITLE
Add extract media url service to media extractor

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add extract media url service to media extractor

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/100780
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/media_extractor.markdown [hunk 1]
```diff
@@ -77,6 +77,15 @@ This will download the file from the given URL.
 | `media_content_id`     | no       | The ID of the content to play. Platform dependent.                                                        |
 | `media_content_type`   | no       | The type of the content to play. Must be one of MUSIC, TVSHOW, VIDEO, EPISODE, CHANNEL or PLAYLIST MUSIC. |
 
+### Extract media url service
+
+There is also a service that will directly return the URL in the response of the service.
+
+| Service data attribute | Optional | Description                                                                                               |
+|-----------------------| -------- |-----------------------------------------------------------------------------------------------------------|
+| `url`                 | no       | The url of the media to extract.                                                                          |
```
Comment (c0ffeeca7) [inline source/_integrations/media_extractor.markdown]:
```suggestion
| `url`                 | no       | The URL of the media to extract.                                                                          |
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @joostlek :+1:

File: source/_integrations/media_extractor.markdown [hunk 2]
```diff
@@ -77,6 +77,15 @@ This will download the file from the given URL.
 | `media_content_id`     | no       | The ID of the content to play. Platform dependent.                                                        |
 | `media_content_type`   | no       | The type of the content to play. Must be one of MUSIC, TVSHOW, VIDEO, EPISODE, CHANNEL or PLAYLIST MUSIC. |
 
+### Extract media url service
```
Comment (c0ffeeca7) [inline source/_integrations/media_extractor.markdown]:
```suggestion
### Extract media URL service
```

**Review by c0ffeeca7 — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Nice 👍

**Pull request merged**